client/my-sites/plugins/plugin-details.jsx

```diff
@@ -545,15 +330,15 @@ function PluginPlaceholder() {
 		<MainComponent wideLayout>
 			<div className="plugin-details__page">
 				<div className="plugin-details__layout plugin-details__top-section is-placeholder">
-					<div className="plugin-details__layout-col plugin-details__layout-col-left">
+					<div className="plugin-details__layout-col-left">
 						<div className="plugin-details__tags">...</div>
 						<div className="plugin-details__header">
 							<div className="plugin-details__name">...</div>
 							<div className="plugin-details__description">...</div>
 							<div className="plugin-details__additional-info">...</div>
 						</div>
 					</div>
-					<div className="plugin-details__layout-col plugin-details__layout-col-right">
+					<div className="plugin-details__layout-col-right">
```

Comment:
There is a regression in the placeholder now. Boxes do not appear the same and also there are some `...` appearing briefly.


|Production | This PR|
|-------|------|
|![SS 2021-12-07 at 13 04 09](https://user-images.githubusercontent.com/12430020/145018002-e6eee345-d9e6-49a4-81fd-5cc2d5fc1a8a.gif)|![SS 2021-12-07 at 13 05 13](https://user-images.githubusercontent.com/12430020/145018117-fa058f4a-9dea-4400-9aca-e90fb563182a.gif)|

Reply:
Thanks for catching this, I'll have a look now.

Reply:
done 99821cd8671373aa48c7fe8eb1746e825281832b